pkg/nuctl/command/deploy.go

```diff
@@ -221,8 +227,11 @@ func newDeployCommandeer(ctx context.Context, rootCommandeer *RootCommandeer, be
 
 	addDeployFlags(cmd, commandeer)
 	cmd.Flags().BoolVarP(&commandeer.skipSpecCleanup, "skip-spec-cleanup", "", false, "Do not clean up spec in function configs")
+	cmd.Flags().BoolVarP(&commandeer.redeployWithReportFile, "redeploy-with-report-file", "", false, "Redeploy any functions that failed in the report and can be redeployed")
+	cmd.Flags().BoolVarP(&commandeer.noReport, "no-report", "", false, "Do not save report to a file")
```

Comment:
I think we should **not** create the report file by default.
Let's make the simplest redeployment command as the default.

so this will be changed to:
```suggestion
	cmd.Flags().BoolVarP(&commandeer.saveReport, "save-report", "", false, "Save redeployment report to a file")
```